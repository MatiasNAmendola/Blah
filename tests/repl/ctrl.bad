24,,13
12,
32
,
,43

if True then
fi

if 12 + 32, 2 then 32 fi
if 12 + then 23 fi

a: 4
if 4 then, 5
fi

if b: 6 then 13
,fi

if False then 54, 23, fi
if 0 then othewise 2 fi
if 0 then else if 2 then 3 othewise 4 fi
if 0 then 1 else if 2 then othewise 4 fi
if 0 then 1 else if 2 then 3 othewise fi

if then
    True
fi

if Nothing then
    True
else
    False
fi
