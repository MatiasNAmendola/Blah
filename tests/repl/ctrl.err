^<parse>
^<parse>
^32$
^<parse>
^<parse>
^<parse>
^<parse>
^<parse>
^<parse>
^<parse>
^<parse>
^<parse>
^<parse>
^<parse>
^<parse>
^<parse>
^<parse>
