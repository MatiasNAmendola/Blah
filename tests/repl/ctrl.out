<parse> [Dk COMMA,Dk COMMA,Dl (Ls (Se (Oa (Ac (Ca (At (Tf (Fp (Ni 24)))))))))] [INT 13]
<parse> [Dk COMMA,Dl (Ls (Se (Oa (Ac (Ca (At (Tf (Fp (Ni 12)))))))))] []
32
<parse> [Dk COMMA] []
<parse> [Dk COMMA] [INT 43]
<parse> [Dk FI,Dk THEN,Do (Oa (Ac (Ca (At (Tf (Fb True)))))),Dk IF] []
<parse> [Dk COMMA,Do (Oa (Ac (Ca (Add (At (Tf (Fp (Ni 12)))) (Tf (Fp (Ni 32))))))),Dk IF] [INT 2,THEN,INT 32,FI]
<parse> [Dk THEN,Dk PLUS,Da (At (Tf (Fp (Ni 12)))),Dk IF] [INT 23,FI]
<parse> [Dk COMMA,Dk THEN,Do (Oa (Ac (Ca (At (Tf (Fp (Ni 4))))))),Dk IF] [INT 5]
<parse> [Dk FI] []
<parse> [Dl (Ls (Assn "b" (Oa (Ac (Ca (At (Tf (Fp (Ni 6))))))))),Dk IF] [THEN,INT 13]
<parse> [Dk COMMA] [FI]
<parse> [Dk FI,Dk COMMA,Dl (Lm (Ls (Se (Oa (Ac (Ca (At (Tf (Fp (Ni 54))))))))) (Se (Oa (Ac (Ca (At (Tf (Fp (Ni 23))))))))),Dk THEN,Do (Oa (Ac (Ca (At (Tf (Fb False)))))),Dk IF] []
<parse> [Dl (Ls (Se (Oa (Ac (Ca (At (Tf (Fp (Ni 2))))))))),Dl (Ls (Se (Oa (Ac (Ca (At (Tf (Fp (Nd "othewise"))))))))),Dk THEN,Do (Oa (Ac (Ca (At (Tf (Fp (Ni 0))))))),Dk IF] [FI]
<parse> [Dk ELSE,Dk THEN,Do (Oa (Ac (Ca (At (Tf (Fp (Ni 0))))))),Dk IF] [IF,INT 2,THEN,INT 3,ID "othewise",INT 4,FI]
<parse> [Dl (Ls (Se (Oa (Ac (Ca (At (Tf (Fp (Ni 4))))))))),Dl (Ls (Se (Oa (Ac (Ca (At (Tf (Fp (Nd "othewise"))))))))),Dk THEN,Do (Oa (Ac (Ca (At (Tf (Fp (Ni 2))))))),Dk IF,Dk ELSE,Dl (Ls (Se (Oa (Ac (Ca (At (Tf (Fp (Ni 1))))))))),Dk THEN,Do (Oa (Ac (Ca (At (Tf (Fp (Ni 0))))))),Dk IF] [FI]
<parse> [Dl (Ls (Se (Oa (Ac (Ca (At (Tf (Fp (Nd "othewise"))))))))),Dl (Ls (Se (Oa (Ac (Ca (At (Tf (Fp (Ni 3))))))))),Dk THEN,Do (Oa (Ac (Ca (At (Tf (Fp (Ni 2))))))),Dk IF,Dk ELSE,Dl (Ls (Se (Oa (Ac (Ca (At (Tf (Fp (Ni 1))))))))),Dk THEN,Do (Oa (Ac (Ca (At (Tf (Fp (Ni 0))))))),Dk IF] [FI]
<parse> [Dk THEN,Dk IF] []
True
<parse> [Dk FI] []
<parse> [Dl (Ls (Se (Oa (Ac (Ca (At (Tf (Fb False)))))))),Dk ELSE,Dl (Ls (Se (Oa (Ac (Ca (At (Tf (Fb True)))))))),Dk THEN,Do (Oa (Ac (Ca (At (Tf (Fp (Nd "Nothing"))))))),Dk IF] []
<parse> [Dk FI] []
